pkg/bindings/images/images.go

```diff
@@ -143,6 +143,7 @@ func Export(ctx context.Context, nameOrIDs []string, w io.Writer, options *Expor
 	for _, ref := range nameOrIDs {
 		params.Add("references", ref)
 	}
+	params.Add("remote", "true")
```

Comment:
This parameter shouldn't be needed as it's always remote.

Reply:
Should I remove all the remote parameter?

Reply:
Should I remove all the remote parameter in all files?

Reply:
Thanks, yes please remove it.